src/ucp/core/ucp_mm.inl

```diff
@@ -43,6 +44,9 @@ ucp_memh_get(ucp_context_h context, void *address, size_t length,
         memh = ucs_derived_of(rregion, ucp_mem_t);
 
         if (ucs_likely(ucs_test_all_flags(memh->md_map, reg_md_map))) {
+            ucs_trace("memh %p: address %p length %zu md_map %" PRIx64
```

Comment:
* let's print in addition to address and length ucp_memh_address(memh)/length(memh) 
* let's add same print in [ucp_memh_get_slow()](https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_mm.c#L433)